uhyve-interface/src/lib.rs

```diff
@@ -0,0 +1,112 @@
+//! # Uhyve Hypervisor Interface
+//!
+//! The uhyve hypercall interface works as follows:
```

Comment:
```suggestion
//! The Uhyve hypercall interface works as follows:
```

I know, this is quite inconsistent at the moment, but in sentences names should be capitalized.

Reply:
But the name "uhyve" is written with a lowercase 'u'. https://github.com/hermitcore/uhyve

Reply:
(Yes, it is inconsistent...)

Reply:
I opened an issue to discuss this separately 👍  https://github.com/hermitcore/uhyve/issues/458